build_tools/python/e2e_test_artifacts/cmake_generator/iree_rule_generator_test.py

```diff
@@ -134,9 +139,13 @@ def test_generate_rules(self):
         entry_function="predict",
         input_types=["2xf32"])
     imported_model_a = iree_definitions.ImportedModel(
-        model=model_a, dialect_type=iree_definitions.MLIRDialectType.TOSA)
+        model=model_a,
+        dialect_type=iree_definitions.MLIRDialectType.TOSA,
+        import_flags="")
     imported_model_b = iree_definitions.ImportedModel(
-        model=model_b, dialect_type=iree_definitions.MLIRDialectType.LINALG)
+        model=model_b,
+        dialect_type=iree_definitions.MLIRDialectType.LINALG,
+        import_flags="--tf-savedmodel-exported-names=predict")
```

Comment:
If having default value on the `import_flags`, we don't need to set it for unrelated one.